proofs/hol_light/arm/README.md

```diff
@@ -33,6 +33,18 @@ make -C proofs/hol_light/arm
 
 will build and run the proofs. Note that this make take hours even on powerful machines.
 
+### macOS (AArch64)
+
+If you want run the proofs from an AArch64 Apple machine, you need to manually install `gobjcopy` via
+
+```
+brew install binutils
+```
+
+and put its parent directory (e.g. `/opt/homebrew/Cellar/binutils/{VERSION}/bin`) into your `PATH`.
```

Comment:
Same comment - probably better to specify
```
/opt/homebrew/opt/binutils/bin
```
to avoid the version number.

Reply:
Done!